app/code/Magento/Cms/Model/ResourceModel/Page.php

```diff
@@ -265,6 +273,28 @@ protected function isValidPageIdentifier(AbstractModel $object)
         return preg_match('/^[a-z0-9][a-z0-9_\/-]+(\.[a-z0-9_-]+)?$/', $object->getData('identifier'));
     }
 
+    /**
+     * Chech whether page custom theme from date is
+     * lower than custom theme to date
+     *
+     * @param AbstractModel $object
+     * @return bool
+     */
+    private function isValidPageCustomThemeDateRange(AbstractModel $object)
```

Comment:
One more thing, should we clarify the type hinting here by using page model instead fo AbstractModel?

Reply:
Hi @rogyar, I've changed it to the CMS Page. Thank you!